ci_framework/roles/switch_config/tasks/ios_config.yml

```diff
@@ -0,0 +1,114 @@
+---
+- name: Set login banner
+  ios_banner:
```

Comment:
Ansible modules all across should be fully-qualified collection names (fqcn).
It is also needed to add these custom modules to the requirements.yml

Reply:
done